CLI tools/CurseForge to Packwiz-Modrinth.py

```diff
@@ -90,32 +95,41 @@ def main() -> int:
             toml.dump(pack_toml, f)
 
         # Packwiz import
-        os.system(packwiz_exe_path + " curseforge import \"" + cf_zip_path + "\"")
+        subprocess.call(f'{packwiz_exe_path} curseforge import "{cf_zip_path}"', shell=True)
         if hydrogen:
-            os.system(packwiz_exe_path + " remove hydrogen")
-            os.system(packwiz_exe_path + " mr install hydrogen")
+            os.system(f"{packwiz_exe_path} remove hydrogen")
+            os.system(f"{packwiz_exe_path} mr install hydrogen")
         if modrinth_overrides:
-            os.system(packwiz_exe_path + " remove entityculling")
-            os.system(packwiz_exe_path + " mr install entityculling")
-            
-    elif refresh_only: 
-        os.system(packwiz_exe_path + " refresh")
+            os.system(f"{packwiz_exe_path} remove entityculling")
+            os.system(f"{packwiz_exe_path} mr install entityculling")
+
+    elif refresh_only:
+        subprocess.call(f"{packwiz_exe_path} refresh", shell=True)
 
     # Copy fresh manifest/modlist to git
     if not is_legacy and not refresh_only:
         extract_file(cf_zip_path, "manifest.json", git_path + "CurseForge", "CurseForge manifest.json", "Git")
         extract_file(cf_zip_path, "modlist.html", git_path + "CurseForge", "CurseForge modlist.html", "Git")
 
-    # Export packwiz pack via mmc-export method
+    # Export Packwiz pack via mmc-export method
     if mmc_export_packwiz_export and not refresh_only:
         mmc_zip_root = str(Path(cf_zip_path).parents[0])
         mmc_zip_path = mmc_zip_root + "\\Fabulously Optimized " + pack_version + ".zip"
         packwiz_config = git_path + "Packwiz\\mmc-export.toml"
 
-        cmd = f'mmc-export -i "{mmc_zip_path}" -f packwiz --modrinth-search loose -o "{mmc_zip_root}" -c "{packwiz_config}" -v {pack_version} --provider-priority Modrinth CurseForge Other --scheme mmc_export_packwiz_output'
-        os.system(cmd)
-
-        packwiz_zip_path = Path(mmc_zip_root) / "mmc_export_packwiz_output.zip" 
+        args = (
+            "mmc-export",
+            "-i", mmc_zip_path,
+            "-f", "packwiz",
+            "--modrinth-search", "loose",
+            "-o", mmc_zip_root,
+            "-c", packwiz_config,
+            "-v", pack_version,
+            "--provider-priority", "Modrinth", "CurseForge", "Other",
+            "--scheme", "mmc_export_packwiz_output",
+        ); subprocess.call(args, shell=True)

```

Comment:
Any particular reason we use `;` here?

Reply:
You don't like it? It's for the sake of compactness mostly and grouping related things together

Reply:
I mean, it makes some sense, but I was commenting on it since I don't usually see it used in anything that's not one line

Reply:
And... What do you want from me exactly? I can remove it if you don't like how it looks

Reply:
I thought there was something against using `;`, but it doesn't seem like it. I don't have the ability to resolve this though, so I guess this stays open until someone can resolve it.

Reply:
Well, the only thing against it is PEP8, since it's "not pythonic"
Direct quote from PEP8 docs: 
```Compound statements (multiple statements on the same line) are generally discouraged.```
So it's just a matter of style choice, I use semicolons when I see appropriate, that's all